pkg/appgw/backendaddresspools.go

```diff
@@ -8,12 +8,14 @@ package appgw
 import (
 	"fmt"
 	"sort"
+	"strings"
 
 	n "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
 	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/golang/glog"
 	v1 "k8s.io/api/core/v1"
 
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/brownfield"
```

Comment:
Too keep functions related to the brownfield deployment feature separate, at least for the time being, I created a new package.  Semantically, these could and should perhaps live within the `backend_pools` go code. Perhaps even `backend_pools` should be a package by itself.  But that conversation and refactoring we could do once this feature here is done.